packages/utilities/src/scroll.scss

```diff
@@ -0,0 +1,3 @@
+.scrollDisabled {
```

Comment:
This converts into a hashed css class, so that if we version it in the future we don't stomp on other versions. CSS classes are a terrible contract. We do this because we've seen partners do things like "if we import utilities, we can add `ms-scroll-body-disabled` class when we want to and it will do what we think. Then we change the classname or what's defined in the class and they break. So obfuscating (making local) the classnames protects us from this implicit dependency.